src/raster/rasterband.rs

```diff
@@ -847,6 +848,87 @@ impl<'a> RasterBand<'a> {
             max: min_max[1],
         })
     }
+
+    /// Fetch default raster histogram.
+    ///
+    /// # Arguments
+    ///
+    /// * `force` - If `true`, force the computation. If `false` and no default histogram is available, the method will return None
+    pub fn default_histogram(&self, force: bool) -> Result<Option<Histogram>> {
```

Comment:
Can we document what returning `Ok(None)` means?

Reply:
In terms of the documentation, I have this in the docs for the `force` parameter:

```rust
/// * `force` - If `true`, force the computation. If `false` and no default histogram is available, the method will return `Ok(None)`
```

Is that enough or you prefer a Note or something similar?

Also, I just realised that histogram should not return None (`GDALGetRasterHistogramEx` does not emit a `CplErrType::Warning`).